go.mod

```diff
@@ -4,15 +4,15 @@ go 1.13
 
 require (
 	github.com/Masterminds/semver v1.5.0 // indirect
```

Comment:
github.com/Masterminds/semver/v3 v3.1.0

Reply:
Updating the dependency to 3.1.0 still brings back 1.5.0. Since it's indirect, I don't see the usage in this code base, but I'm guessing the module using it is using `github.com/Masterminds/semver` instead of `github.com/Masterminds/semver/v3`